core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -135,6 +151,19 @@ public void configure() throws ConfigurationException {
 	public void open() throws FileSystemException {
 		super.open();
 		basefolderId = getBaseFolderId(getMailAddress(),getBaseFolder());
+
```

Comment:
Ik denk dat regel 153 onder het onderstaande if statement geplaatst moet worden.

In getBaseFolderId wordt er immers al gebruik gemaakt van een exchangeService.

Reply:
ja eens